### ISSUE TITLE
About Page: Our Mission and Our Values Allignment

### ISSUE THREAD

**Issue by pxh1432** [girldevelopit/gdi-website#320]
The our mission and our values paragraphs take a while for me to decipher because of the alignment. 
Instead of having them right next to each other can we place the paragraphs on top of each other and still keep the center alignment like the rest of the page. I feel it would it would connect better and reduce the visual confusion.

```
                                                        [ Our Vision ]
```

Our vision is to create a network of empowered women who feel confident in their abilities to code and build beautiful web and mobile applications. By teaching women around the world from diverse backgrounds to learn software development, we can help women improve their careers and confidence in their everyday lives.

```
                                                        [ Our Values ]
```

We are committed to making sure women of all races, education levels, income and upbringing can build confidence in their skill set to develop web and mobile applications. Our goal is to provide powerful hands-on programs to women seeking professional help in software development and create basic to advanced web and mobile applications.

**Comment by kstack7** [comment]
Any designers/UXers want to share their thoughts on this one?

**Comment by kstack7** [comment]
@Cara-Jo @Roenok @corinnepw thoughts?

**Comment by Cara-Jo** [comment]
I agree with these comments and say :shipit: with these suggestions.

**Comment by kstack7** [comment]
Anyone willing to try this out? Just swapping the Vision and Values from the side-by-side layout to a stacked look.

**Comment by shanirivers** [comment]
Hi, I'm willing to work on this.

**Comment by kstack7** [comment]
hey @johnmarinelli can you post your screenshot here? We typically discuss these changes as a group and then create the pull requests that reflect the decision. We'd appreciate it!

**Comment by johnmarinelli** [comment]
no problem:
![gdi](https://cloud.githubusercontent.com/assets/3382949/8576633/4fbdf700-2558-11e5-8040-3b6fd7e03f9d.png)

**Comment by kstack7** [comment]
Ello folks!

I just pushed @johnmarinelli's changes to staging so you can see them here. I will push them to production this week unless I hear any reasons not to. Thanks again, @johnmarinelli!

https://girl-develop-it-staging.herokuapp.com/about

**Comment by kstack7** [comment]
Fixed with #353
